tests/Microsoft.DotNet.Framework.Docker.Tests/ImageTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.Framework.Docker.Tests
+{
+    public class ImageTests
+    {
+        private const string WSC2016 = "windowsservercore-ltsc2016";
```

Comment:
Consider renaming `WSC2016` to `WSC_LTSC`

Reply:
I don't think WSC_LTSC is sufficient because there will be other LTSC versions over time therefore this would be ambiguous.  Changed to `WSC_LTSC2016` and `WSC_1709`